holdem/player.py

```diff
@@ -93,20 +97,20 @@ def update_localstate(self, table_state):
   def player_move(self, table_state, action):
     self.update_localstate(table_state)
     bigblind = table_state.get('bigblind')
-    tocall = min(table_state.get('tocall', 0), self.stack)
+    tocall = min(table_state.get('tocall', 0), self.stack + self.currentbet)
```

Comment:
`self.stack + self.currentbet` shows up a lot, might be worth extracting a variable here.

Reply:
Sure, just wanted to go over bugs first. Planning to improve the environment a lot.

Reply:
Done!